bastion.tf

```diff
@@ -26,16 +26,10 @@ resource "equinix_metal_port" "bastion_bond0" {
   port_id         = [for p in equinix_metal_device.bastion.ports : p.id if p.name == "bond0"][0]
   layer2          = false
   bonded          = true
-  vlan_ids        = [data.equinix_metal_vlan.bastion.vlan_id]
+  vlan_ids        = [module.metal_vrf_gateways_w_dynamic_neighbor["bastion"].vlan_uuid]
```

Comment:
I don't see where this `"bastion"` key is set up.  Does that already exist somewhere in the main branch?

Reply:
Main.tf does a `for each` of the `vcf_vrf_networks` variable and instantiates a `module.metal_vrf_gateways_w_dynamic_neighbor` for each one.